store/rootmulti/store.go

```diff
@@ -757,8 +757,17 @@ func (rs *Store) Snapshot(height uint64, protoWriter protoio.Writer) error {
 	// and the following messages contain a SnapshotNode (i.e. an ExportNode). Store changes
 	// are demarcated by new SnapshotStore items.
 	for _, store := range stores {
+		fmt.Printf("Snapshot.Export Store Key %s\n", store.name)
```

Comment:
we should use the logger and place this in debug mode.

Reply:
will fix tomorrow.  didn't see the logger earlier, but I see it now